packages/injectable/src/getInjectable/getInjectable.js

```diff
@@ -1,5 +1,12 @@
 import identity from 'lodash/fp/identity';
+import lifecycleEnum from '../dependency-injection-container/lifecycleEnum';
 
 // Note: this function exists only for typed presence in TypeScript.
 // It has little purpose in JavaScript.
-export default identity;
+export default function getInjectable(rawInjectable) {
```

Comment:
Why do we need this? The public product of `getInjectable()` should not need this information?